backend/inspirehep/assign/views.py

```diff
@@ -82,3 +84,37 @@ def assign_view():
     if to_author_recid is None:
         return jsonify({"stub_author_id": stub_author_id}), 200
     return jsonify({"message": "Success"}), 200
+
+
+@blueprint.route("literature_conferences", methods=["POST"])
+@login_required_with_roles([Roles.cataloger.value])
+def literature_assign_conferences_view():
+    body = request.get_json()
+    conference_recid = body.get("conference_recid")
+    literature_recids = body.get("literature_recids")
+
+    if not conference_recid or not literature_recids:
```

Comment:
Instead I would use `@parser.use_args` decorator to check the args

Reply:
:+1: